### ISSUE TITLE
Add wallet indication for tokens that the user hold

### ISSUE THREAD

**Issue by oznasi1** [bancorprotocol/carbon-app#299]
### My Wallet indication
please add another list option that shows the tokens i hold in the wallet.
when connecting wallet -> fetch user token balance and display the tokens and their balances.
sort:
- fetch token balances * cryptoCompare.USD.Value -> highest value to lowest.

IF wallet not connected -> do not show this option
IF wallet is empty -> show empty list
IF token doesnt have usd value -> show it **after** the tokens with value. sort based on token amounts (if more than 1)

**Comment by ashachaf** [comment]
@pingustar how expensive and is there a way to achieve something like this cheaper?

**Comment by pingustar** [comment]
@ashachaf in terms of quota being used for the alchemy api ... I have no idea can do some testing if you want.

Multicall with 5000 tokens also might take a few seconds to do, we might even need to split it up into multiple calls. Something like batching it in 1000 token buckets. Which we then could fetch in parallel to speed things up.

Can definitely do some testing on this.

We need to keep in mind that this would be polled at some interval and after a transaction is done. Technically we can only update the affected token after a tx. But polling would still be needed to stay in sync.

> @pingustar how expensive and is there a way to achieve something like this cheaper?
